platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -238,6 +238,9 @@ dependencies {
 }
 ```
 
+#### Introduced Isolated Projects safe project path accessor on `ProjectDependency`
```

Comment:
I'm not sure we want to mention Isolated Projects here

cc @alllex

Reply:
Correct. So far, we have decided that the introduction of IP in the release notes will be more comprehensive and intentional. We won't do it based on related, but smaller changes.